src/invite/InviteAndSendModal.tsx

```diff
@@ -33,7 +33,9 @@ export default function InviteAndSendModal({ isVisible, name, onInvite, onCancel
       image={inviteModal}
       testID="InviteAndSendModal"
     >
-      {t('inviteAndSendDialog.body', { name, amount: rewardAmount, currency })}
```

Comment:
@lisaqyli @MuckT FYI i'm updating this since this would be a breaking change (relevant to our discussion in eng sync) since older versions of the app would not have access to amount of currency, they would see broken copy once OTA translations are published :)

i am getting around this by introducing this new copy only for users eligible for invite rewards, since this modal is shown to everyone so we shouldn't mention rewards in case they are not eligible